aws/aws-how-to/instances/automatically-setup-multiple-nics.rst

```diff
@@ -0,0 +1,55 @@
+Set up automated network configuration for instances with multiple NICs
```

Comment:
Is it okay if we change the title to "Automatically configure multiple NICs"?

Reply:
Done.